backend/src/main/java/com/pickpick/slackevent/application/message/MessageCreatedService.java

```diff
@@ -75,8 +75,7 @@ private Channel createChannel(final String channelSlackId) {
                     request -> request.channel(channelSlackId)
             ).getChannel();
 
-            Channel channel = toChannel(conversation);
-            channels.save(channel);
+            Channel channel = channels.save(toChannel(conversation));
 
             return channel;
```

Comment:
```suggestion
            Channel channel = toChannel(conversation);
            return channels.save(channel);
```

이런 방식은 어떨까요?
저장되는 대상이 무엇인지 명시적으로 드러낼 수 있을 것 같아요.